firebase/package.json

```diff
@@ -42,6 +43,7 @@
     "prettier": "^2.4.x",
     "sinon": "^10.0.x",
     "ts-node": "^10.2.x",
+    "typescript": "^4.5.x",
```

Comment:
Adds missing typescript dependency to the project (we have installed as a global package in our machines, but the CI hasn't)